### ISSUE TITLE
[A11y] Explore Collections - Table - Table headers

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1904]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

The collection names should be set as row headers.  Column headers are already good.

### Suggested Fix

Use `<th>` with `scope="row"` for all cells that should be announced when moving from row to row

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Likely needs expertise - NVDA makes this trivial to test, but if you're very careful and detailed you can verify the rows are tagged correctly.

Make sure when moving from one row to the next, e.g. if you're in the "INSTITUTION" column, that you hear the collection announced.
